play/dbt-materialize/License.md

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I think I can get rid of this, but want to confirm before I do!

Reply:
Did you write all of the code yourself or did you copy the code?

Reply:
Indeed, @JLDLaughlin, there is no need to include another copy of the Apache License in the repository, since we've already got a copy in the LICENSE file at the root of the repository. But we do need to replace the BSL headers on each of the files you've imported with Apache 2.0 headers:

```py
# Copyright 2020 Josh Wills. All rights reserved.
# Copyright Materialize, Inc. All rights reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License in the LICENSE file at the
# root of this repository, or online at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

@jwills could you please confirm that you are the copyright holder for your original contribution?

Reply:
Indeed I am.